### ISSUE TITLE
In Minecraft: Wii U Edition, worlds don't delete properly

### ISSUE THREAD

**Issue by spudpiggy** [cemu-project/Cemu#606]
I am on version 2.0-25. I don't know where to find the commit hash but this issue also occurs in 1.27.1.
Within the emulator, deleting a world does not seem to work. It just stays on the world screen.
Minecraft Wii U stores worlds in two files: one without an extension, and a .ext file.
I just tried to delete a world in Cemu, and once it failed I went to the save directory to manually delete it.
But I noticed something strange. The extensionless file actually was deleted! This file contains world data. I don't know, but I highly suspect that the .ext file provides world metadata. It seems like the game will display a world so long as its .ext file is present.

TL;DR On 2.0-25 (and also 1.27.1, probably others too), you can't delete worlds from within Minecraft Wii U. It deletes the world data but not the associated .ext file, so it's still listed. Attached is a log, although it doesn't seem helpful. (I'm clueless but it looks way too short to be of any use. I've gotta be doing something wrong.)
[log.txt](https://github.com/cemu-project/Cemu/files/10366244/log.txt)
